Project.toml

```
@@ -17,5 +18,6 @@ HDF5 = "0.14, 0.15"
 ProgressMeter = "1.3"
 StaticArrays = "0.12, 1.0"
 TimerOutputs = "0.5"
+Trixi = "0.3.30"
```

```suggestion
Trixi = "0.3.31"
```
to get rid of `n_cells_max`